packages/protobuf/src/message.ts

```diff
@@ -141,8 +141,15 @@ export class Message<T extends Message<T> = AnyMessage> {
    * unpacked, and this is only possible with a type registry to look up the
    * message type.  As a result, attempting to serialize a message with this
    * type will throw an Error.
+   *
+   * Note that this function is private because users should not need to invoke
+   * it directly -- they can use JSON.stringify.  Additionally, if it is public
+   * it shows as part of the Message API to users which can be confusing when
+   * listed with the other toJson method
```

Comment:
It's now "protected", no longer private. We should call it "method" instead of "function". Last sentence is missing a full stop.

Reply:
Ugh sorry.  That is my fault.